### PR TITLE
bomradarcam camera platform documentation

### PR DESCRIPTION
**Pull request by maddenp**
**Description:**

Documentation for a proposed `bomradarcam` (Australian Bureau of Meteorology radar-imagery loop) camera platform.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#21183

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/camera.bomradarloop.markdown [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+title: "BOM Radar Loop"
+description: "Instructions for integrating bomradarloop within Home Assistant"
+date: 2019-02-18 00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Camera
+logo: home-assistant.png
+ha_release: 0.88
```
Comment (klaasnicolaas) [inline source/_components/camera.bomradarloop.markdown]: 0.89 or later

File: source/_components/camera.bomradarloop.markdown [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+layout: page
+title: "BOM Radar Loop"
+description: "Instructions for integrating bomradarloop within Home Assistant"
+date: 2019-02-18 00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Camera
+logo: home-assistant.png
```
Comment (klaasnicolaas) [inline source/_components/camera.bomradarloop.markdown]: Please use a logo of the component and not of Home Assistant

File: source/_components/camera.bomradarloop.markdown [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+
+## {% linkable_title Configuration %}
+
+To enable `bomradarloop` in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+camera:
+  - platform: bomradarloop
+    location: yourlocation
```
Comment (klaasnicolaas) [inline source/_components/camera.bomradarloop.markdown]:
```suggestion
    location: YOUR_LOCATION
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/camera.bomradarloop.markdown (hunk 2 above)
Comment (klaasnicolaas) [reply source/_components/camera.bomradarloop.markdown]:
```suggestion
logo: bom.png
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/camera.bomradarloop.markdown (hunk 1 above)
Comment (maddenp) [reply source/_components/camera.bomradarloop.markdown]: Updated to 0.89.

File: source/_components/camera.bomradarloop.markdown (hunk 2 above)
Comment (maddenp) [reply source/_components/camera.bomradarloop.markdown]: Updated to use `bom.png`.

File: source/_components/camera.bomradarloop.markdown (hunk 3 above)
Comment (maddenp) [reply source/_components/camera.bomradarloop.markdown]: Fixed, thank you -- that was in the instructions and I overlooked it.

**Comment by maddenp** [comment]
Updated platform name in docs due to refactor in corresponding `home-assistant` PR.

**Comment by frenck** [comment]
Closing this PR, since the parent PR has been closed.

**Pull request closed without merging**